README

```diff
@@ -34,6 +34,56 @@ xtensa-byt-elf-objcopy -I binary -O elf32-xtensa-le -B xtensa image.bin image.bi
 
 xtensa-byt-elf-objdump -D image.bin.elf > image.dis.txt
 
+
+logger
+======
+
+logger is used to print logs delivered from FW dma_trace mechanism, by searching log
+entries in *.ldc file generated by rimage
```

Comment:
Thanks for the documentation! Could you also give a brief introduction about *.ldc file?  Or is there already a document about *.ldc file?

Reply:
@mengdonglin I've updated that.